clients/src/main/java/org/apache/kafka/common/metrics/ClientTelemetryPayload.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.common.metrics;
+
+import org.apache.kafka.common.Uuid;
+import org.apache.kafka.common.annotation.InterfaceStability;
+
+import java.nio.ByteBuffer;
+
+@InterfaceStability.Evolving
+public interface ClientTelemetryPayload {
+
+    /**
+     * Client's instance id.
+     */
+    Uuid clientInstanceId();
+
+    /**
+     * Indicates whether client is terminating, e.g., the last metrics push from this client instance.
+     */
+    boolean isTerminating();
+
+    /**
+     * Metrics data content-type / serialization format.
+     * Currently "application/x-protobuf;type=otlp+metrics0.11"
```

Comment:
We decided that if/when the client supports other content types that we'd augment the code. Right now, the `PushTelemetryRequestData` doesn't include a content type. It seems inconsistent to have methods that imply support for other content types on the broker.

Not a PR blocker, but just curious why it's still like this in the KIP only on the broker side.

Reply:
@kirktrue I have just copied this interface from the KIP-714 however i still see no harm as the current implementation returns OTLP as the default type